actor-testkit-typed/src/test/java/jdocs/org/apache/pekko/actor/testkit/typed/javadsl/LogCapturingExampleTest.java

```diff
@@ -26,7 +26,7 @@
 
 public class LogCapturingExampleTest {
 
-  @ClassRule public static final TestKitJunitResource testKit = new TestKitJunitResource();
+  @ClassRule public static final TestKitJUnitResource testKit = new TestKitJUnitResource();
```

Comment:
Can we leave the legacy Junit 4 classes alone?

I must also admit to preferring that the new classes are `Junit5` to match the pre-existing standard. I am going to -1 the PR due to the renaming of the existing classes.

Reply:
ok i changed it back :-)

Reply:
I disagree, I think the class should be named `JUnit5` and we should have a deprecated alias for `Junit5`.